src/js/components/search/SearchOption.jsx

```diff
@@ -39,27 +40,32 @@ export default class SearchOption extends React.Component {
     }
 
     render() {
+        let searchOption = null;
         switch (this.props.name) {
             case 'Keywords':
-                return (<div className="search-option">
-                    <SearchBox
-                        toggleFilter={this.toggleFilter.bind(this)}
-                        showFilter={this.state.showFilter}
-                        arrowState={this.state.arrowState} />
-                </div>);
-            case 'AwardType':
-                return (<div className="search-option"><AwardTypeContainer
-                    toggleFilter={this.toggleFilter.bind(this)}
-                    showFilter={this.state.showFilter}
-                    arrowState={this.state.arrowState} /></div>);
-            case 'TimePeriod':
-                return (<div className="search-option"><TimePeriod
-                    toggleFilter={this.toggleFilter.bind(this)}
-                    showFilter={this.state.showFilter}
-                    arrowState={this.state.arrowState} /></div>);
+                searchOption = (<SearchBox
+                    showFilter={this.state.showFilter} />);
+                break;
+            case 'Award Type':
+                searchOption = (<AwardTypeContainer
+                    showFilter={this.state.showFilter} />);
+                break;
+            case 'Time Period':
+                searchOption = (<TimePeriod
+                    showFilter={this.state.showFilter} />);
+                break;
             default:
-                return (<div className="search-option"><b>{this.props.name}</b></div>);
+                searchOption = null;
         }
+
+        return (<div className="search-option">
```

Comment:
Add some line breaks between the parens to match our standard coding style for component `render` return statements.